src/language/providers/utils.ts

```diff
@@ -283,6 +291,33 @@ export function checkSetVariableOpen(
   return false
 }
 
+export function getNsPrefix(
+  document: vscode.TextDocument,
+  position: vscode.Position
+) {
+  var initialLineNum = position.line
+  var lineNum = 0
+  while (initialLineNum !== 0 && lineNum <= initialLineNum) {
+    const lineText = document
+      .lineAt(lineNum)
+      .text.substr(0, document.lineAt(lineNum).range.end.character)
```

Comment:
Doesn't this substring just getting the whole line? Why not just use this to get the whole line:

```js
const lineText = document.lineAt(lineNum).text`
```

Reply:
document.lineAt() returns a TextLine.  The includes() property isn't available on a TextLine, It is available on a string, which is what is returned by the proprty text.substr().
![image](https://user-images.githubusercontent.com/98881959/190625588-c7d36410-33df-4111-bd4a-d443a58042ae.png)

Reply:
but `TextLine` has a `.text` property, the [TextLine doc](https://code.visualstudio.com/api/references/vscode-api#TextLine) says:

> text: string
>
> The text of this line without the line separator characters.

So calling `document.lineAt(lineNum).text` should get the entire line as a string.

Reply:
I don't think this comment was adressed? Everything else looks good to me.